templates/_includes/series.html

```diff
@@ -10,7 +10,7 @@ <h4>{{ article.series.index }}</h4>
          {% for part_article in article.series.all %}
             <li {% if part_article == article %} class="active-part"{% endif %}>
             {% if part_article != article %}
-            <a href="{{ SITEURL }}/{{ part_article.url }}" title="{{ part_article.title }}">Part {{ article.series.all.index(part_article)+1 }}: {{ part_article.title }}</a>
+            <a href="{{ SITEURL }}/{{ part_article.url }}" title="{{ part_article.title|striptags|e }}">Part {{ article.series.all.index(part_article)+1 }}: {{ part_article.title }}</a>
             {% else %}
```

Comment:
@lzimd shouldn't the we also use `striptags` in the text inside `<a>` tag?

Reply:
With typogrify, it will be a challenge.

```html
 <a href="/foo" title="bar">
    Part 2: <span class="caps">bar</span>
 </a>
```

will be

```html
<a href="/foo" title="<span class ="caps"="">bar"&gt;
    Part 2: <span class="caps">bar</span>
</a>
```

because of typogrify's `<span class ="caps">bar</span>`.

Reply:
Ok thanks for the clarification.